docs/content/docs/dev/table/materialized-table/statements.md

```diff
@@ -165,15 +165,15 @@ FRESHNESS = INTERVAL '5' HOUR
 ```sql
 -- The refresh mode of the created materialized table is CONTINUOUS, and the job's checkpoint interval is 1 hour.
 CREATE MATERIALIZED TABLE my_materialized_table
-    REFRESH_MODE = CONTINUOUS
     FRESHNESS = INTERVAL '1' HOUR
+    REFRESH_MODE = CONTINUOUS
```

Comment:
what is the purpose of this change - it seems to just move a line. I assume it works before.